GameData/RP-1/Parts/Science/CR01_5-CherenkovCounter.cfg

```diff
@@ -0,0 +1,52 @@
+//**********************************************************************************
+//  Cherenkov Detector CRT
+//
+//	Based on the Cherenkov Counter Cosmic Ray Telescopes used on many spacecraft
+//  for more detailed study of the Cosmic Energy particles.
+//
+//	Part originally created by CobaltWolf as the Pioneer 10 Cosmic Ray Telescope
+//
+//**********************************************************************************
+
+PART
+{
+	name = RP0Cherenkov
+	module = Part
+	author = SierraHotel
+	
+	RSSROConfig = True
+	
+	MODEL
+	{
+		model = RP-1/Parts/Science/Assets/bluedog_geiger
+	}
+	scale = 1.0
+	rescaleFactor = 1.2
+	node_attach = 0.0, 0.0, 0.0, 0.0, -1.0 , 0.0
+	TechRequired = basicScience
+	entryCost = 0
+	cost = 250
+	category = Science
+	subcategory = 0
+	title = CR01.5 - Cherenkov Cosmic Ray Detector
```

Comment:
This should just be CD01 I think, Cherenkov Detector, because it's not actually in the line of cosmic ray detectors, it's a separate instrument for us

Reply:
Seems so obvious now...